server/src/main/java/org/elasticsearch/index/seqno/GlobalCheckpointSyncAction.java

```diff
@@ -62,7 +63,7 @@ public GlobalCheckpointSyncAction(
             actionFilters,
             Request::new,
             Request::new,
-            ThreadPool.Names.MANAGEMENT
+            ThreadPool.Names.WRITE
```

Comment:
I think we need to force execution of this action, because the `WRITE` threadpool queue is still bounded and yet it's important we don't drop these syncs.

Reply:
Yes sorry I missed that in this PR.